bin/hive_setup

```diff
@@ -196,12 +196,12 @@ def setup_hive(options)
     f.puts ''
     f.puts '  # Logging configuration'
     f.puts '  logging:'
-    f.puts "    directory: #{dir}/log"
-    f.puts "    pids: #{dir}/pids"
+    f.puts "    directory: <%=ENV['HIVE_RUNNER_PATH'] || '#{dir}' %>/log"
```

Comment:
Instead of `<%=ENV['HIVE_RUNNER_PATH'] || '#{dir}' %>`, we can do something like `ENV['HIVE_RUNNER_PATH'] || '#{dir}'` But why do we want to set `HIVE_RUNNER_PATH` from env?